ipa-core/src/report.rs

```diff
@@ -413,6 +434,186 @@ where
     }
 }
 
+/// A binary report as submitted by a report collector, containing encrypted `OprfReport`
+/// An `EncryptedOprfReport` consists of:
+///     `ct_mk`: Enc(`match_key`)
+///     `ct_btt`: Enc(`breakdown_key`, `trigger_value`, `timestamp`)
+///     associated data of `ct_mk`: `key_id`, `epoch`, `event_type`, `site_domain`,
+
+#[derive(Copy, Clone, Eq, PartialEq)]
+pub struct EncryptedOprfReport<BK, TV, TS, B>
```

Comment:
I would strongly recommend to add the report structure to the description similar to IPA report

```rust
// Report structure:
//  * 0..4: `timestamp`
//  * 4: `breakdown_key`
//  * 5..a: `trigger_value`
//  * a..b: `encap_key`
//  * b..c: `mk_ciphertext`
//  * c: `event_type`
//  * c+1: `key_id`
//  * c+2..c+4: `epoch`
//  * c+4..: `site_domain`
```

reading through the code to understand report structure is not ideal